bands/site_sections/band_admin.py

```diff
@@ -24,6 +24,21 @@ def mark_as_band(self, session, group_id):
             'message': '{} has been marked as a band'.format(group.name)
         }
 
+    @ajax
+    def remove_as_band(self, session, group_id):
+        group = session.group(group_id)
+        if not group.leader:
+            return {'message': '{} does not have an assigned group leader'.format(group.name)}
```

Comment:
Yeah, this could cause problems since the system would let you make a leaderless group a band but not undo it without adding a leader.